Tests/DependencyInjection/DoctrineMigrationsExtensionTest.php

```diff
@@ -277,6 +277,71 @@ public function testPrefersEntityManagerOverConnection() : void
         self::assertSame($em, $di->getEntityManager());
     }
 
+    /**
+     * @return array<string[]|null[]>
```

Comment:
```suggestion
     * @return array<array{string|null}>
```

Reply:
i've tried this, but PHPCS complains... this was the only syntax i've found that was making happy both phpstan and phpcs

Reply:
but let me re try...

Reply:
tried again, phpcs does not like it..

Reply:
That might be because we use that super ancient version of our CS: https://github.com/doctrine/DoctrineMigrationsBundle/blob/aa15026788c2f5bfc4df899dfcd4d4612022fd1e/composer.json#L31

Reply:
See #365

Reply:
done